lib/entries.rb

```diff
@@ -1,15 +1,79 @@
 require 'pry'
+require_relative 'symbols.rb'
 
 class Entries
 
-  def initialize(account_number)
-    @account_number
-    @account_number_display = []
+  def initialize
+    @symbols = Symbols.new
+    @entry = 0
+    @entries_as_symbols = [[],[],[]]
+    @account_numbers = []
   end
 
-  attr_accessor :account_number
+  attr_accessor :entry, :account_numbers
+  attr_reader :symbols, :entries_as_symbols
 
-  def display_digits(account_number)
-    account_number_array = account_number.split('')
+
+# code to validate entry before passing it to symbols.rb - does not work.
+  # def entry_validation(entry)
+  #   entry_number_of_digits(entry)
+  #   true
+  # rescue
+  #   puts 'Exception intercepted. 9 digit number required.'
+  #   false
+  # end
+  #
+  # def entry_number_of_digits(entry)
+  #   return true if entry.entry.to_s.chars.map(&:to_i).count != 9
+  # end
+
+#User story 3 solution
+  def print_account_numbers
+    ac_nmb = get_account_numbers
```

Comment:
I'd try to avoid variable names like this as they're not descriptive enough. `account_numbers` feels like a better name